plugin/yt_dlp_plugins/extractor/getpot_bgutil_script.py

```diff
@@ -70,10 +70,10 @@ def _get_pot(self, client: str, ydl: YoutubeDL, visitor_data=None, data_sync_id=
             raise RequestError(
                 f'_get_pot_via_script failed: Unable to run script (caused by {e!s})') from e
 
-        self._logger.debug(f'stdout = {stdout}')
+        self._logger.debug(f'stdout:\n{stdout}\nstderr:\n{stderr}')
         if returncode:
             raise RequestError(
-                f'_get_pot_via_script failed with returncode {returncode}:\n{stderr.strip()}')
+                f'_get_pot_via_script failed with returncode {returncode}')
 
         # The JSON response is always the last line
         script_data_resp = stdout.splitlines()[-1]
```

Comment:
Line 80: maybe more useful if we log the `json.loads`'ed response?

Reply:
is there any big difference? printing the raw json string vs however python stringifies its dictionary objects?

Reply:
the raw json string should be in the stdout above already.
an example showing the difference:
```log
[debug] [GetPOT] BgUtilScriptPot: stdout:
{"visitIdentifier":"CgtaMU1xN3lTbUdSZyjcmpm3BjIKCgJBVRIEGgAgFg%3D%3D","poToken":"MniMfshIJMIyQoxCtYapTA1Wt1oxnn4fsxffqVA25ugaXNhtVU4-Teb5zXb9c4QVsM9TYJWUxddAaWTqU6aqswTF1PxcuCI-qb-AxdipvSotaB8s4uLLe3aW8hsP9R6smLQGLTmSSJ05cF8qwop-w2pGIUCcNn3nb8g=","generatedAt":"2024-09-15T02:58:39.194Z"}
[debug] [GetPOT] BgUtilScriptPot: _get_pot_via_script response = {'visitIdentifier': 'CgtaMU1xN3lTbUdSZyjcmpm3BjIKCgJBVRIEGgAgFg%3D%3D', 'poToken': 'MniMfshIJMIyQoxCtYapTA1Wt1oxnn4fsxffqVA25ugaXNhtVU4-Teb5zXb9c4QVsM9TYJWUxddAaWTqU6aqswTF1PxcuCI-qb-AxdipvSotaB8s4uLLe3aW8hsP9R6smLQGLTmSSJ05cF8qwop-w2pGIUCcNn3nb8g=', 'generatedAt': '2024-09-15T02:58:39.194Z'}
```
the differences arent big, though